lib/helpers/conferences.rb

```diff
@@ -108,6 +108,7 @@ def conferences
       {
         '/2021-losangeles/' => 'Overview',
         '/2021-losangeles/registration/' => 'Registration',
+        '/2021-losangeles/submit/' => 'CfP',
```

Comment:
In the past we've called this "Submit a talk". Is the motivation for "CfP" the menu space issue? Anyway, fine for now, but I'd prefer to not rely on people having to know the abbreviation.

Reply:
This is a copy and paste from last year